### PR TITLE
Redirect Remote access API from wiki to jenkins.io

### PR DESCRIPTION
**Pull request by zaycodes**

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]
This is the most requested page on wiki.jenkins.io based on web access logs collected for the past 90 days.  Thanks very much @zaycodes !

File: dist/profile/templates/confluence/vhost.conf [hunk 1]
```diff
@@ -3003,6 +3003,8 @@ RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/m
 RewriteRule "^/display/JENKINS/FreeBSD$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/FreeBSD\+4.9$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+report\+an\+issue$" "https://www.jenkins.io/participate/report-issue/" [NE,NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
```
Comment (olblak) [inline dist/profile/templates/confluence/vhost.conf]:
```suggestion
```
This line is duplicated

**Review by olblak — CHANGES_REQUESTED** [review]
for some reason, I can't commit my suggestion, so if you could remove the duplicated rewriteCond

**Comment by zaycodes** [comment]
> This is the most requested page on wiki.jenkins.io based on web access logs collected for the past 90 days. Thanks very much @zaycodes !

You are welcome @MarkEWaite

**Pull request merged**